tests/antares/services/local_services/test_area.py

```diff
@@ -1174,3 +1175,15 @@ def test_translation_txt_is_empty_by_default(self, area_fr, fr_load):
 
         # Then
         assert actual_file_contents == expected_file_contents
+
+
+class TestReadArea:
+    def test_read_areas_local(self, local_study_with_hydro):
```

Comment:
You could use _w_areas instead of _with_hydro as it produces the same result and it's a simpler study

Reply:
understood! done with next commit